File: Domaine/Entities/Promotion.cs
namespace Domaine.Entities
{
    public class Promotion
    {
        public Guid PromotionID { get; set; }
        public string Code { get; set; }
        public decimal Discount { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public ICollection<Product> Product { get; set; }

    }
}

File: ParaAPP/src/Types/Promotion.ts
export interface CreatePromotion {
  code: string;
  discount: number;
  startDate: string;
  endDate: string;
  productIds: string[];
}
export interface Promotion {
  promotionID: string;
  code: string;
  discount: number;
  startDate: string;
  endDate: string;
  productIds: string[];
}
